Bedford-PD/bedford_complaints.yaml

```diff
@@ -0,0 +1,543 @@
+    src: "person"
+    dst: "hearing"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["COMPL_TYPE_CD", "DEFENDANT_MSTR_NO"]
+    name: "appearsin"
+
+  subjectof4:
+    fqn: "ol.subjectof"
+    entitySetName: "MiddlesexBedfordSubjectOf"
+    src: "person"
+    dst: "summons"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["COMPL_TYPE_CD", "DEFENDANT_MSTR_NO"]
+    name: "subjectof4"
+
+  has:
+    fqn: "ol.has"
+    entitySetName: "MiddlesexBedfordHas"
+    src: "person"
+    dst: "persondetails"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["DEFENDANT_MSTR_NO", "JUVENILE_IND"]
+    name: "has"
+
+  involvedin1:
+    fqn: "ol.involvedin"
+    entitySetName: "MiddlesexBedfordInvolvedIn"
+    src: "person"
+    dst: "incident"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["DEFENDANT_MSTR_NO", "COMPL_NO"]
+    name: "involvedin1"
+
+  involvedin2:
+    fqn: "ol.involvedin"
+    entitySetName: "MiddlesexBedfordInvolvedIn"
+    src: "employee"
+    dst: "incident"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["COMPL_EMPL_NO", "COMPL_NO"]
+    name: "involvedin2"
+
+  relatedto1:
+    fqn: "ol.relatedto"
+    entitySetName: "MiddlesexBedfordRelatedTo"
+    src: "arrest"
+    dst: "offenseevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMPL_TYPE_CD"
+    name: "relatedto1"
+
+  relatedto2:
+    fqn: "ol.relatedto"
+    entitySetName: "MiddlesexBedfordRelatedTo"
+    src: "warrant"
+    dst: "offenseevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMPL_TYPE_CD"
+    name: "relatedto2"
+
+  relatedto3:
+    fqn: "ol.relatedto"
+    entitySetName: "MiddlesexBedfordRelatedTo"
+    src: "summons"
+    dst: "offenseevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMPL_TYPE_CD"
+    name: "relatedto3"
+
+  relatedto4:
+    fqn: "ol.relatedto"
+    entitySetName: "MiddlesexBedfordRelatedTo"
+    src: "hearing"
+    dst: "offenseevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMPL_TYPE_CD"
+    name: "relatedto4"
+
+  clearedby:
+    fqn: "ol.clearedby"
+    entitySetName: "MiddlesexBedfordClearedBy"
+    src: "offenseevent"
+    dst: "disposition"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["ONLY_MISDEMEANORS_IND", "ONE_OR_MORE_FELONIES_IND", "COMMENTS"]
+    name: "clearedby"
+
+  partof:
```

Comment:
Should this be related to?
(this is what we decided for Dane)

Reply:
partof is definitely better. How many integrations already have relatedto for this? If only like 1 or 2, then I think it's worth reintegrating them with partof. It will only get harder to make that change.